src/chrome/content/rules/University-of-South-Florida.xml

```diff
@@ -1,68 +1,51 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://usfsp.edu/ => https://usfsp.edu/: (51, "SSL: no alternative certificate subject name matches target host name 'usfsp.edu'")
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://usfsp.edu/ => https://usfsp.edu/: (51, "SSL: no alternative certificate subject name matches target host name 'usfsp.edu'")
-	For problematic rules, see University-of-South-Florida-mismatches.xml.
-
-
-	Nonfunctional usf.edu subdomains:
```

Comment:
Please take care that you're not removing any information that is still valid.

Reply:
I wasn't aware that I did? I removed rules that do/will not have valid SSL protection on them or that are no longer in production/do not exist/are not maintained. 

The usfsp.edu no SAN one in particular will not be resolved.

Reply:
E.g. `system.usf.edu` is still a a non-functional target. We use to document them to make sure that we haven't missed them: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

Reply:
Good point... Ok, I guess I'll work on documenting those and resub a new PR later.